### PR TITLE
Custom Notification Dot Colors

### PR DESCRIPTION
**Pull request by yasanglass**
Adds custom color support to Notification Dots. This is a part of [this pull request on the main lawnchair repository](https://github.com/LawnchairLauncher/lawnchair/pull/2789).

See the other pull request for more information.

### CODE REVIEW

File: iconloaderlib/src/com/android/launcher3/icons/DotRenderer.java [hunk 1]
```diff
@@ -137,14 +139,22 @@ public void draw(Canvas canvas, DrawParams params, int numNotifications) {
         canvas.translate(dotCenterX + offsetX, dotCenterY + offsetY);
         canvas.scale(params.scale, params.scale);
 
+        // Color
+        int dotColor;
+        if (mColor != -1) {
```
Comment (suphon-t) [inline iconloaderlib/src/com/android/launcher3/icons/DotRenderer.java]: Let's use `0` instead of this, since white is `-1`.

Comment (paphonb) [reply iconloaderlib/src/com/android/launcher3/icons/DotRenderer.java:144]: Actually, we could just use `0`.

Comment (yasanglass) [reply iconloaderlib/src/com/android/launcher3/icons/DotRenderer.java]: I have set the value of `AppIcon` to `-1`. Isn't `0` black?

Comment (suphon-t) [reply iconloaderlib/src/com/android/launcher3/icons/DotRenderer.java]: No. 0 is transparent.

Comment (yasanglass) [reply iconloaderlib/src/com/android/launcher3/icons/DotRenderer.java]: Ill change both of the values to 0 then.

**Review by suphon-t — APPROVED** [review]

**Pull request merged**